tests/mongodb/_connection/run.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+MONGOPORT=22824
+rm -f log.txt*
+rm -rf db
+mkdir -p db/noauth
+mkdir -p db/wcert
+mkdir -p db/auth
+
+if ! eval $DUB_INVOKE -- $MONGOPORT failconnect ; then
+	exit 1
+fi
+
+# Unauthenticated Test
+
+MONGOPID=$(mongod --logpath log.txt --bind_ip 127.0.0.1 --port $MONGOPORT --noauth --dbpath db/noauth --fork | grep -Po 'forked process: \K\d+')
```

Comment:
Why not just do this?

```
mongod --logpath log.txt --bind_ip 127.0.0.1 --port $MONGOPORT --noauth --dbpath db/noauth &
MONGOPID=$?
```

Reply:
the `--fork` makes sure the server is fully started before it continues script execution

Reply:
added comment why not